pkg/scaffold/v2/crd/enablewebhook_patch.go

```diff
@@ -36,7 +36,7 @@ type EnableWebhookPatch struct {
 // GetInput implements input.File
 func (p *EnableWebhookPatch) GetInput() (input.Input, error) {
 	if p.Path == "" {
-		p.Path = filepath.Join("config", "crds", "patches",
+		p.Path = filepath.Join("config", "crd", "patches",
```

Comment:
Is there a convention to use singular or plural?

Reply:
it was by default just the name of the generator in CT.  Since we're overriding it anyway, it can be whatever